### PR TITLE
Merge changes from Images version of juicy-filedrop.html #3

### PR DESCRIPTION
**Pull request by aloscha**

### CODE REVIEW

**Comment by aloscha** [comment]
It clear previous file name after choose another file

File: starcounter-upload.html [hunk 1]
```diff
@@ -69,7 +69,9 @@
                 return !autoUpload && tasksLength > 0;
             },
             selectFile: function () {
-                this.$.fileElement.click();
+				var element = this.$.fileElement;
+                element.value = "";
+                element.click();
             },
```
Comment (warpech) [inline starcounter-upload.html:75]: Too much whitespace on line 72. Please fix and merge to gh-pages.

**Comment by warpech** [comment]
Just one comment, please see.

File: starcounter-upload.html (hunk 1 above)
Comment (aloscha) [reply starcounter-upload.html:75]: Sorry @warpech I've used Notepad++ instead of VS

**Comment by aloscha** [comment]
Fixed

**Comment by warpech** [comment]
Thanks. I will release a version.

**Pull request merged**